### PR TITLE
update our readme

### LINKED ISSUE #38659: Our Readme is out of date
Our repo's readme.md is in need of updates.  I noticed several things recently:

- We only list some of our dependent repos.
- We should update it for the new GitHub feedback control on docs.
- We should describe our expectations for issues and PRs more clearly. This should include our general response times on PR reviews, issues. Also, include how to request updated reviews.
- We should direct readers to our project boards to track our work and comment.

---
[Associated WorkItem - 190728](https://dev.azure.com/msft-skilling/Content/_workitems/edit/190728)

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #38659

Periodic update to our readme to match our current processes.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [issues-policy.md](https://github.com/dotnet/docs/blob/e8f89e64ce8781f2d10125589889aeb7419de480/issues-policy.md) | [GitHub issues process and policy](https://review.learn.microsoft.com/en-us/dotnet/issues-policy?branch=pr-en-us-38674) |

### CODE REVIEW

File: issues-policy.md [hunk 1]
```diff
@@ -65,9 +65,7 @@ Priority levels are based on the following guidelines but are subjective. The mi
 
 Team members spend a limited amount of time on diagnosis and triage so they can make progress on scheduled tasks. Each team member spends at most 30 minutes per day in diagnosis and triage.
 
-The **up-for-grabs** label is applied when an issue is a good candidate for a community member (possibly the author) to submit a fix. The team member that applies the **up-for-grabs** label will help or find someone to help community members work through the PR creation process. Issues that are "up for grabs" are often added to the [Community Contribution projects](https://github.com/dotnet/docs/projects/35)
-
-> NOTE: We've only recently adopted the preceding convention. The person that added the label may refer you to another team member who will help.
+The **help-wanted** label is applied when an issue is a good candidate for a community member (possibly the author) to submit a fix. The team member that applies the **help-wanted** label will help or find someone to help community members work through the PR creation process. Issues that are "up for grabs" are often added to the [Community Contribution projects](https://github.com/dotnet/docs/projects/35).
```
Comment (gewarren) [inline issues-policy.md:68]:
```suggestion
The **help-wanted** label is applied when an issue is a good candidate for a community member (possibly the author) to submit a fix. The team member that applies the **help-wanted** label will help or find someone to help community members work through the PR creation process. These kinds of issues are often added to the [Community Contribution projects](https://github.com/dotnet/docs/projects/35).
```

File: README.md [hunk 2]
```diff
@@ -10,10 +10,14 @@
...
+Our teams tasks are tracked in our [project boards](https://github.com/dotnet/docs/projects?query=is%3Aopen). You'll see monthly sprint projects, along with long-running projects for major documentation updates. The projects contain documentation issues across the repositories that build .NET docs. Issues are tracked in the relevant repositories. We have a large community using these resources. We make our best effort to respond to issues in a timely fashion. You can read more about our procedures for classifying and resolving issues in our [Issues policy](issues-policy.md) topic. To create a new issue, click the "Open documentation issue" button at the bottom of any of our published docs, or [choose from any of the available templates](https://github.com/dotnet/docs/issues/new/choose). The control at the bottom of each article automatically routes you to the correct repo and fills in some relevant information based on the article.
```
Comment (gewarren) [inline README.md:20]:
```suggestion
Our team's tasks are tracked in our [project boards](https://github.com/dotnet/docs/projects?query=is%3Aopen). You'll see monthly sprint projects, along with long-running projects for major documentation updates. The projects contain documentation issues across the repositories that build .NET docs. Issues are tracked in the relevant repositories. We have a large community using these resources. We make our best effort to respond to issues in a timely fashion. You can read more about our procedures for classifying and resolving issues in our [Issues policy](issues-policy.md) topic. To create a new issue, click the "Open documentation issue" button at the bottom of any of our published docs, or [choose from one of the available templates](https://github.com/dotnet/docs/issues/new/choose). The control at the bottom of each article automatically routes you to the correct repo and fills in some relevant information based on the article.
```

**Review by gewarren — APPROVED** [review]

**Review by IEvangelist — APPROVED** [review]
After Genevieve's suggestions, LGTM.

**Review by tdykstra — APPROVED** [review]
I left a typo fix suggestion and a question about a link.

File: issues-policy.md [hunk 3]
```diff
@@ -65,9 +65,7 @@ Priority levels are based on the following guidelines but are subjective. The mi
 
 Team members spend a limited amount of time on diagnosis and triage so they can make progress on scheduled tasks. Each team member spends at most 30 minutes per day in diagnosis and triage.
 
-The **up-for-grabs** label is applied when an issue is a good candidate for a community member (possibly the author) to submit a fix. The team member that applies the **up-for-grabs** label will help or find someone to help community members work through the PR creation process. Issues that are "up for grabs" are often added to the [Community Contribution projects](https://github.com/dotnet/docs/projects/35)
-
-> NOTE: We've only recently adopted the preceding convention. The person that added the label may refer you to another team member who will help.
+The **help-wanted** label is applied when an issue is a good candidate for a community member (possibly the author) to submit a fix. The team member that applies the **help-wanted** label will help or find someone to help community members work through the PR creation process. These kinds of issues are often added to the [Community Contribution projects](https://github.com/dotnet/docs/projects/35).
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply issues-policy.md:68]: Great catch. I'd removed that reference from the readme, but missed it in this file.

**Pull request merged**